File: OPENAPI-REST-API/swagger-client/dart/lib/model/affiliate_traffic_row.dart
part of swagger.api;

class AffiliateTrafficRow {
  
  String trafficId = null;

  String trafficIp = null;

  String trafficUrl = null;

  String trafficAffiliate = null;

  String trafficReferrer = null;

  String trafficTimestamp = null;

  AffiliateTrafficRow();

  @override
  String toString() {
    return 'AffiliateTrafficRow[trafficId=$trafficId, trafficIp=$trafficIp, trafficUrl=$trafficUrl, trafficAffiliate=$trafficAffiliate, trafficReferrer=$trafficReferrer, trafficTimestamp=$trafficTimestamp, ]';
  }

  AffiliateTrafficRow.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    trafficId = json['traffic_id'];
    trafficIp = json['traffic_ip'];
    trafficUrl = json['traffic_url'];
    trafficAffiliate = json['traffic_affiliate'];
    trafficReferrer = json['traffic_referrer'];
    trafficTimestamp = json['traffic_timestamp'];
  }

  Map<String, dynamic> toJson() {
    return {
      'traffic_id': trafficId,
      'traffic_ip': trafficIp,
      'traffic_url': trafficUrl,
      'traffic_affiliate': trafficAffiliate,
      'traffic_referrer': trafficReferrer,
      'traffic_timestamp': trafficTimestamp
     };
  }

  static List<AffiliateTrafficRow> listFromJson(List<dynamic> json) {
    return json == null ? new List<AffiliateTrafficRow>() : json.map((value) => new AffiliateTrafficRow.fromJson(value)).toList();
  }

  static Map<String, AffiliateTrafficRow> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, AffiliateTrafficRow>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new AffiliateTrafficRow.fromJson(value));
    }
    return map;
  }
}

File: OPENAPI-REST-API/swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/myadmin-client-csharp-dotnet2/Model/AffiliateTrafficRow.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// Affiliate Web Traffic Entry
  /// </summary>
  [DataContract]
  public class AffiliateTrafficRow {
    /// <summary>
    /// Gets or Sets TrafficId
    /// </summary>
    [DataMember(Name="traffic_id", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "traffic_id")]
    public string TrafficId { get; set; }

    /// <summary>
    /// Gets or Sets TrafficIp
    /// </summary>
    [DataMember(Name="traffic_ip", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "traffic_ip")]
    public string TrafficIp { get; set; }

    /// <summary>
    /// Gets or Sets TrafficUrl
    /// </summary>
    [DataMember(Name="traffic_url", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "traffic_url")]
    public string TrafficUrl { get; set; }

    /// <summary>
    /// Gets or Sets TrafficAffiliate
    /// </summary>
    [DataMember(Name="traffic_affiliate", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "traffic_affiliate")]
    public string TrafficAffiliate { get; set; }

    /// <summary>
    /// Gets or Sets TrafficReferrer
    /// </summary>
    [DataMember(Name="traffic_referrer", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "traffic_referrer")]
    public string TrafficReferrer { get; set; }

    /// <summary>
    /// Gets or Sets TrafficTimestamp
    /// </summary>
    [DataMember(Name="traffic_timestamp", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "traffic_timestamp")]
    public string TrafficTimestamp { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class AffiliateTrafficRow {\n");
      sb.Append("  TrafficId: ").Append(TrafficId).Append("\n");
      sb.Append("  TrafficIp: ").Append(TrafficIp).Append("\n");
      sb.Append("  TrafficUrl: ").Append(TrafficUrl).Append("\n");
      sb.Append("  TrafficAffiliate: ").Append(TrafficAffiliate).Append("\n");
      sb.Append("  TrafficReferrer: ").Append(TrafficReferrer).Append("\n");
      sb.Append("  TrafficTimestamp: ").Append(TrafficTimestamp).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
